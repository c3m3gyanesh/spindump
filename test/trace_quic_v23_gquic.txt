A HTTP/3 test between Picoquic client and the Chromium "gQUIC" server implementation. The command that was used:

    ./picoquic/picoquicdemo -l ./test.log -L -a "h3-23" -v ff000017 quic.rocks 4433 /50000
