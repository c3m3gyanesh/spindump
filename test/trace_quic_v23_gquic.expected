QUIC 172.16.114.251 <-> 216.155.158.183 085cb13ea03bd2f9-000d7c29148e7563 (61188:4433) at 1571182209984000 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 216.155.158.183 085cb13ea03bd2f9-08000d7c29148e75 (61188:4433) at 1571182210099737 measurement up right 115737 packets 1 0 bytes 1280 0 bandwidth 1280 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 216.155.158.183 5cb13ea03bd2f9ca-08000d7c29148e75 (61188:4433) at 1571182210121264 spinvalue up 0 initiator packets 4 4 bytes 1554 5120 bandwidth 1554 5120 note "V23,no spin"
QUIC 172.16.114.251 <-> 216.155.158.183 5cb13ea03bd2f9ca-08000d7c29148e75 (61188:4433) at 1571182210121297 spinvalue up 0 initiator packets 5 4 bytes 1658 5120 bandwidth 1658 5120 note "V23,no spin"
QUIC 172.16.114.251 <-> 216.155.158.183 5cb13ea03bd2f9ca-08000d7c29148e75 (61188:4433) at 1571182210194388 spinvalue up 0 responder packets 6 5 bytes 3158 5198 bandwidth 3158 5198 note "V23,no spin"
QUIC 172.16.114.251 <-> 216.155.158.183 5cb13ea03bd2f9ca-08000d7c29148e75 (61188:4433) at 1571182210194673 spinvalue up 1 initiator packets 6 6 bytes 3158 6478 bandwidth 3158 6478 note "V23,no spin"
QUIC 172.16.114.251 <-> 216.155.158.183 5cb13ea03bd2f9ca-08000d7c29148e75 (61188:4433) at 1571182210204696 spinvalue up 0 responder packets 7 6 bytes 3241 6478 bandwidth 3241 6478 note "V23,no R-spin"
QUIC 172.16.114.251 <-> 216.155.158.183 5cb13ea03bd2f9ca-08000d7c29148e75 (61188:4433) at 1571182210204955 spinvalue up 1 initiator packets 7 7 bytes 3241 6561 bandwidth 3241 6561 note "V23,no R-spin"
QUIC 172.16.114.251 <-> 216.155.158.183 5cb13ea03bd2f9ca-08000d7c29148e75 (61188:4433) at 1571182210224400 spinvalue up 0 responder packets 8 7 bytes 3324 6561 bandwidth 3324 6561 note "V23,no R-spin"
QUIC 172.16.114.251 <-> 216.155.158.183 5cb13ea03bd2f9ca-08000d7c29148e75 (61188:4433) at 1571182210224821 spinvalue up 1 initiator packets 8 8 bytes 3324 6659 bandwidth 3324 6659 note "V23,no R-spin"
QUIC 172.16.114.251 <-> 216.155.158.183 5cb13ea03bd2f9ca-08000d7c29148e75 (61188:4433) at 1571182210234364 spinvalue up 0 responder packets 9 9 bytes 3389 6736 bandwidth 3389 6736 note "V23,no R-spin"
QUIC 172.16.114.251 <-> 216.155.158.183 5cb13ea03bd2f9ca-08000d7c29148e75 (61188:4433) at 1571182210234613 spinvalue up 1 initiator packets 9 10 bytes 3389 6797 bandwidth 3389 6797 note "V23,no R-spin"
QUIC 172.16.114.251 <-> 216.155.158.183 5cb13ea03bd2f9ca-08000d7c29148e75 (61188:4433) at 1571182210305659 spinvalue up 1 responder packets 10 10 bytes 3449 6797 bandwidth 3449 6797 note "V23,no R-spin"
QUIC 172.16.114.251 <-> 216.155.158.183 5cb13ea03bd2f9ca-08000d7c29148e75 (61188:4433) at 1571182210305659 measurement up right 110986 packets 10 10 bytes 3449 6797 bandwidth 3449 6797 note "V23,spinning"
